### PR TITLE
Update Production Package Testing to 1.6.0

### PR DESCRIPTION
**Pull request by fundthmcalculus**

### CODE REVIEW

File: python/trinsic/__main__.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import asyncio
-import platform
+from asyncio import set_event_loop_policy
```
Comment (geel9) [inline python/trinsic/__main__.py]:
```suggestion
from trinsic.trinsic_util import set_event_loop_policy
```

**Review by geel9 — CHANGES_REQUESTED** [review]

**Comment by fundthmcalculus** [comment]
node will continue to fail for reasons related to: https://github.com/trinsic-id/sdk/issues/838

**Comment by geel9** [comment]
This PR breaks our docs by removing `sample_demo.py`, which is imported here:

https://docs-v2.trinsic.id/python/

Given that we have the python test script that can be executed via CLI, approval for me to commit to this branch removing this injection @fundthmcalculus?

**Comment by fundthmcalculus** [comment]
Go for it. Thanks!

**Review by geel9 — APPROVED** [review]

**Pull request merged**